e2etests/web/regular_integration_tests/test_driver/platform_messages_e2e.dart

```diff
@@ -0,0 +1,58 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:html' as html;
+// ignore: undefined_shown_name
+import 'dart:ui' as ui show platformViewRegistry;
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:regular_integration_tests/platform_messages_main.dart' as app;
+
+import 'package:e2e/e2e.dart';
+
+void main() async {
+  E2EWidgetsFlutterBinding.ensureInitialized() as E2EWidgetsFlutterBinding;
+
+  testWidgets('platform message for Clipboard.set/getData reply with future',
```

Comment:
nit: Rename the test. I think we only tests the setData here.

Reply:
Done.